macros/PG.pl

```diff
@@ -1207,8 +1207,9 @@ sub ENDDOCUMENT {
 						type  => 'button',
 						class => "ww-feedback-btn btn btn-sm $options{btnClass} $options{btnAddClass}"
 						. ($rh_envir->{showMessages} && $ansHash->{ans_message} ? ' with-message' : ''),
-						'aria-label' => $options{resultTitle},
-						data         => {
+						'aria-label' => $options{resultTitle}
+						. ($rh_envir->{showMessages} && $ansHash->{ans_message} ? ' with message' : ''),
```

Comment:
This should be translated.  If you need help with that, let me know.

Reply:
Is it just ` maketext(' with message')`? Or should it encompass the `$options{resultTitle}` too?

Reply:
It should include the title.  So this should be `maketext('[_1] with message', $options{resultTitle})` in the case that there is a message.